src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Elasticsearch/FrameworkRepository.cs

```diff
@@ -107,5 +103,27 @@ public FrameworkCodeSummary GetFrameworkByCode(int frameworkCode)
 
             return document != null ? _frameworkMapping.MapToFrameworkCodeSummary(document) : null;
         }
+
```

Comment:
Is it possible to refactor the code blocks like this to be generic?

`new SearchDescriptor<FrameworkSearchResultsItem>()
 +                .Index(_applicationSettings.ApprenticeshipIndexAlias)
 +                .Type(Types.Parse("frameworkdocument"))
 +                .From(0)
 +                .Sort(sort => sort.Ascending(f => f.FrameworkId))
 +                .Take(take)
 +                .MatchAll()` 

This pattern is in a lot of classes and they seem to look very similar